osgp/platform/osgp-adapter-domain-tariffswitching/src/test/java/org/opensmartgridplatform/adapter/domain/tariffswitching/application/services/ScheduleManagementServiceTest.java

```diff
@@ -0,0 +1,130 @@
+package org.opensmartgridplatform.adapter.domain.tariffswitching.application.services;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.opensmartgridplatform.adapter.domain.tariffswitching.application.mapping.DomainTariffSwitchingMapper;
+import org.opensmartgridplatform.adapter.domain.tariffswitching.infra.jms.core.OsgpCoreRequestMessageSender;
+import org.opensmartgridplatform.domain.core.entities.Device;
+import org.opensmartgridplatform.domain.core.entities.DeviceOutputSetting;
+import org.opensmartgridplatform.domain.core.entities.Ssld;
+import org.opensmartgridplatform.domain.core.repositories.SsldRepository;
+import org.opensmartgridplatform.domain.core.services.DeviceDomainService;
+import org.opensmartgridplatform.domain.core.services.OrganisationDomainService;
+import org.opensmartgridplatform.domain.core.valueobjects.LightValue;
+import org.opensmartgridplatform.domain.core.valueobjects.RelayType;
+import org.opensmartgridplatform.domain.core.valueobjects.ScheduleEntry;
+import org.opensmartgridplatform.shared.exceptionhandling.ComponentType;
+import org.opensmartgridplatform.shared.exceptionhandling.FunctionalException;
+import org.opensmartgridplatform.shared.infra.jms.CorrelationIds;
+import org.opensmartgridplatform.shared.infra.jms.RequestMessage;
+
+@ExtendWith(MockitoExtension.class)
+public class ScheduleManagementServiceTest{
+	@Mock
+	private DeviceDomainService deviceDomainService;
+	
+	@Mock
+	private DomainTariffSwitchingMapper domainCoreMapper;
+	
+	@Mock 
+	private SsldRepository ssldRepository;
+	
+	@Mock
+	private OrganisationDomainService organisationDomainService;
+
+	@Mock
+	private OsgpCoreRequestMessageSender osgpCoreRequestMessageSender;
+
+	@Mock
+	private Device device;
+	
+	@Mock
+	private Ssld ssld;
+	
+	@Mock
+	private CorrelationIds correlationIds;
+	
+	@InjectMocks
+	private ScheduleManagementService scheduleManagementService;
+	
+	private List<ScheduleEntry> scheduleEntries;
+	private List<DeviceOutputSetting> deviceOutputSettings;
+	private List<LightValue> lightValues;
+	private final long scheduleTime = (long) 0.0f;
+	private final String messageType = "testType";
+	private final int priority = 1;
+
+	public void setup() throws FunctionalException {
```

Comment:
method should be private
a method called setup in a test class is usually annotated with BeforeEach
in this test class, the method is only called from one test method, so the name is a bit confusing.
I would name it arrangeSetTariffScheduleTest and insert it after testSetTariffSchedule